This includes a few headers to fix some missing function declarations.

Signed-off-by: Samuel Thibault <samuel.thibault@ens-lyon.org>

diff --git a/stubdom/grub/config.h b/stubdom/grub/config.h
index 1649d51..15a6583 100644
--- a/stubdom/grub/config.h
+++ b/stubdom/grub/config.h
@@ -2,6 +2,7 @@
 #undef putchar
 #include <ctype.h>
 #include <string.h>
+#include <kernel.h>
 #define debug _debug
 #define grub_halt(a) do_exit()
 #define printf grub_printf
diff --git a/xen/include/xen/libelf.h b/xen/include/xen/libelf.h
index 2a6fa54..e65db6d 100644
--- a/xen/include/xen/libelf.h
+++ b/xen/include/xen/libelf.h
@@ -39,11 +39,13 @@ typedef int elf_negerrnoval; /* 0: ok; -EFOO: error */
 #ifdef __XEN__
 #include <public/elfnote.h>
 #include <public/features.h>
+#include <xen/string.h>
 #else
 #include <xen/elfnote.h>
 #include <xen/features.h>
 
 #include <stdarg.h>
+#include <string.h>
 
 struct elf_binary;
 typedef void elf_log_callback(struct elf_binary*, void *caller_data,

